modular_zubbers/code/modules/jobs/stowaway.dm

```diff
@@ -0,0 +1,42 @@
+/datum/job/assistant/stowaway
+	title = "Stowaway"
+	description = "Be a stowaway crewmember."
+	faction = FACTION_NONE
+	supervisors = "Yourself."
+	minimal_player_age = 7
+	exp_requirements = 2400
+	exp_required_type = EXP_TYPE_CREW
+	exp_granted_type = EXP_TYPE_SPECIAL
+
+	liver_traits = list(TRAIT_MAINTENANCE_METABOLISM)
+
+
+	job_flags = JOB_NEW_PLAYER_JOINABLE|JOB_REOPEN_ON_ROUNDSTART_LOSS|JOB_ASSIGN_QUIRKS|JOB_CANNOT_OPEN_SLOTS
+
+/datum/job/assistant/stowaway/after_spawn(mob/living/spawned, client/player_client)
+	. = ..()
+	for(var/obj/item/card/id/to_del in spawned.contents)
+		spawned.dropItemToGround(to_del, force = TRUE)
+		qdel(to_del)
+	for(var/obj/item/modular_computer/pda/to_del in spawned.contents)
+		spawned.dropItemToGround(to_del, force = TRUE)
+		qdel(to_del)
+
+	var/mob/living/carbon/human/human_holder = spawned
+	var/datum/bank_account/account = SSeconomy.bank_accounts_by_id["[human_holder.account_id]"]
+	account?.Destroy()
+	var/turf/T
+	if(LAZYLEN(GLOB.blobstart))
+		var/list/blobstarts = shuffle(GLOB.blobstart)
+		for(var/_T in blobstarts)
+			T = _T
+			break
```

Comment:
You should probably use the build in proc ``find_maintenance_spawn(atmos_sensitive = TRUE, require_darkness = FALSE)`` instead of the custom code here. Blob spawn turfs are meant for blobs who don't need to breathe or worry about temperature, and generally have no guidelines upstream in terms of where they should be placed in terms of safety, since only blobs use them. I'm pretty sure on some maps there are blob spawns inside atmos tanks.

Also fun fact: On metastation, there are some blob spawn locations in erp-heavy areas or extremely sketchy, such as:
- The library backroom.
- Some bathroom stalls.
- The detective's closet.
- Evidence Room storage.

Reply:
you are right

Reply:
:sunglasses: